Apps/W1/EDocumentConnectors/Logiq/app/src/EDocument.PageExt.al

```diff
@@ -0,0 +1,48 @@
+namespace Microsoft.EServices.EDocumentConnector.Logiq;
+
+using Microsoft.eServices.EDocument;
+
+
+pageextension 6380 "E-Document" extends "E-Document"
+{
+    actions
+    {
+        addafter(GetApproval)
+        {
+            action(UpdateStatus)
```

Comment:
Great that we have an action. 👍 🥇 

Set visibility based on if document relates to Service, and is incomming/Outgoing?
Can you change this to use the IDocumentAction interface. Then you dont have to do the manual logging you are doing. You should not do that essentially. Framework shoud. 

I can see that this also checks approval. What is stauses is it checking. Do we have a more specific name?

Reply:
If this is for sent documents. Can we use Approval action that already exissts?